### PR TITLE
[LIBS-230] Add basic implementation

### PR DESCRIPTION
**Pull request by smilin-desperado**

### CODE REVIEW

File: plugin.yml [hunk 1]
```diff
@@ -1,9 +1,47 @@
...
+      description: Enable debug mode
+    pacticipant:
+      type: string
+      description: Name of consumer/provider e.g. service
+    action:
+      type: string
+      enum:
+        - pr
+        - merge
+      description: The type of pipeline
```
Comment (GZLiew) [inline plugin.yml:22]: does it have a default or we will need to specify one?

**Review by GZLiew — APPROVED** [review]
🚀 🧩 LGTM!

File: plugin.yml (hunk 1 above)
Comment (smilin-desperado) [reply plugin.yml:22]: no default but I've created this ticket --> https://benieworldwide.atlassian.net/browse/LIBS-232 to work it out based on the current pipeline name so this option will be an override instead

Comment (GZLiew) [reply plugin.yml:22]: Noice!

**Pull request merged**